manage.py

```diff
@@ -312,3 +316,16 @@ def config(config_key):
 @app.cli.command()
 def set_celery_beat_healthy():
     return celery_beat_health_check()
+
+
+@app.cli.command()
+def healthcheck():
+    """Calls the healthcheck API"""
+    result = requests.get(
+        url_for('check')
+    )
+    print(json.dumps(result.json(), indent=4))
+
+    # Return success (0) if passing status code
```

Comment:
I'd prefer the HTTP code as the exit code (or `0` for 200/OK/success) if it's easy enough.

Reply:
just added this in.